packages/story-editor/src/karma/element-library/text/edit.karma.js

```diff
@@ -149,6 +149,32 @@ describe('TextEdit integration', () => {
           '<span style="font-weight: 700">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>'
         );
       });
+
+      it('should select all text and delete it', async () => {
+        // Testing this on the BROKEN version
+        await fixture.events.mouse.clickOn(frame, 30, 5); // enter the edit mode by clicking
+
+        if (navigator.userAgentData.platform === 'macOS') {
+          document.execCommand('selectAll'); // not the same as mod+a, but does work on macOS
+        } else {
+          await fixture.events.keyboard.shortcut('mod+a'); // doesn't work on macOS, works on Ubuntu
+        }
```

Comment:
```suggestion
        // Needed because of https://github.com/puppeteer/puppeteer/issues/1313
        if (navigator.userAgentData.platform === 'macOS') {
          document.execCommand('selectAll'); // not the same as mod+a, but does work on macOS
        } else {
          await fixture.events.keyboard.shortcut('mod+a'); // doesn't work on macOS, works on Ubuntu
        }
```